src/firebase/firebase_config.js

```diff
@@ -4,12 +4,29 @@ import { getFirestore, collection, getDocs } from 'firebase/firestore/lite';
 import { getDatabase } from "firebase/database";
 import firebase from 'firebase/compat/app'; 
 import "firebase/compat/firestore";
+import "firebase/compat/auth";
```

Comment:
dont think we should be using compat libraries: https://firebase.google.com/docs/web/modular-upgrade#about_the_namespaced_compat_libraries

Reply:
I can take this on as a follow up task, since I was the one who started using the compat first.

Reply:
Seems modular API will be faster and was suggested by Firebase. I will change the compat to the modular API in the next PR.